### DISCUSSION TITLE
ling2db scaffold issue on .NET 8

### DISCUSSION THREAD

**Discussion by mariorancic01** [linq2db/linq2db#4427] [General]
I tried to do this: dotnet linq2db scaffold -i database.json --customize bin\Debug\net8.0\MyDll.dll

and I got this error: Could not load file or assembly 'System.Runtime, Version=8.0.0.0

**Comment by MaceWindu** [comment]
Could you rebuild your custom dll with lower .net version?

**Comment by mariorancic01** [comment]
I just tried to do it with .net 7 and I got message: Cannot find
interceptor class in 'bin\Debug\net7.0\MyDll.dll'

Actually the problem is that I used before Linq2Db.Tools 4.4.0 version on
.net core 3.1 and this worked and i'll stay with that for now. I think
interceptor classes have changed since then.

El mar, 20 feb 2024 a la(s) 12:06 p.m., MaceWindu ***@***.***)
escribió:

> Could you rebuild your custom dll with lower .net version?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/linq2db/linq2db/discussions/4427#discussioncomment-8528448>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG66BVUCTL36AW3ACUQDCM3YUR7UTAVCNFSM6AAAAABDP5RXVWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKMRYGQ2DQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Mario Rancic
Software Engineer
Microsoft Certified Professional

**Reply by MaceWindu** [reply]
Could be. You need to build against same version of linq2db.tools

**Comment by MaceWindu** [comment]
.net 8 support just got released
